tools/release.sh

```diff
@@ -7,7 +7,7 @@ Usage() {
     exit 1
 }
 
-if [ "`dirname $0`" != "tools" ] ; then
+if [ "`dirname $0`" != "./tools" ] ; then
```

Comment:
You can just run it as `tools/release.sh`. You don't need the leading `./`.

If you want to change this so it can be run with the leading dot, perhaps you should check against both `tools` and `./tools` or check that `tools/release.sh` is the same file as `$0`.